### ISSUE TITLE
Font Appearance page crashes the app

### ISSUE THREAD

**Issue by 4aiman** [Danily07/Translumo#7]
I can't change text appearance. 

Did it once, then realized it was a poor choice.
Now it crashes upon every try to change the font color.

Recorded in 0.8, but 0.8.5 has the same issue (got only the update).
Can't attach any logs, since the app haven't produced any.
Launching "as admin" doesn't help.

If there's a way to get more info out of the app - plz do tell me how.

https://github.com/Danily07/Translumo/assets/2913210/0bf98fe8-6155-4849-9a94-a53015615ca1

**Comment by Danily07** [comment]
Hi! Try to delete user settings file (/Users/<User name>/AppData/Roaming/Translumo). But please attach it to the issue before that, maybe I can reproduce the bug

**Comment by 4aiman** [comment]
Deleted the file, but it didn't help (admin and non-admin).
Here's a copy: [settings.zip](https://github.com/Danily07/Translumo/files/11797002/settings.zip)

I'll try a clean install of Translumo meanwhile.

**Comment by Danily07** [comment]
pls also check if there are any related errors in windows event viewer
![ka02R000000i2vcQAA_ru_3](https://github.com/Danily07/Translumo/assets/29047281/62ae4acd-79d5-4182-ae56-c5b8ea0e9edc)

**Comment by 4aiman** [comment]
```
Имя сбойного приложения: Translumo0.8.5.exe, версия: 0.8.5.0, метка времени: 0x62577ba6
Имя сбойного модуля: ntdll.dll, версия: 10.0.22621.1848, метка времени: 0x48d14984
Код исключения: 0xc0000374
Смещение ошибки: 0x000000000010be19
Идентификатор сбойного процесса: 0x0x3588
Время запуска сбойного приложения: 0x0x1D9A3797AA34D86
Путь сбойного приложения: F:\Installers\Translumo-v.0.8.0\Translumo0.8.5.exe
Путь сбойного модуля: C:\Windows\SYSTEM32\ntdll.dll
Идентификатор отчета: e7fbd447-6ac4-47c6-af34-fce7360aaf0b
Полное имя сбойного пакета: 
Код приложения, связанного со сбойным пакетом: 
```

**Comment by Danily07** [comment]
Thanks, Is there only one error in event viewer?  For example, the second one can looks like that
![err](https://github.com/Danily07/Translumo/assets/29047281/87bf3636-dcad-48f6-876d-477dec778257)

- You have Windows 11 installed, right? Now it looks like external system problem, I found quite similar issue, which is related with windows 11 quite random bug occured in .NET WPF/Winforms applications. I didn't get if MS released an update with fix, though...
- Please also attach crach dump file from Users\"User"\AppData\Local\CrashDumps

**Comment by 4aiman** [comment]
Yes, Win11 pro, Build 22621.
And yes, that's the only error event.
I open the settings and it crashes.

*I've fixed it on my end*

Here's how:
 - I didn't have a crash dump that corresponds to the error on the screen, but I've relaunched the app and caused a new crash.
 - I was hesitant to supply a 400+ Mb file to anyone w/o first knowing what I'm supplying, so I've downloaded WinDbg.
 - WinDbg started to load all the relevant files and I've noticed GTIII-OSD among the files.
 - I knew that's OSD from the ASUS GPU Tweak III - an app like MSI Afterburner that shows performance stats for GPU and can be used to fine tune your GPU power/temperature limits.
 - I've disabled the OSD and Translumo stopped crashing in both 0.8 and 0.8.5.

I'm still hesitant to send the dump, but I'm eager to follow any instructions to further test this case of overlay "conflict".
Seeing as you're the only dev, it might be better to offload this issue of mine to me.

In hindsight is seems weird I haven't checked the GPU stuff earlier. Win 11 has known issues with NVIdia GPU drivers, overlays in general, and the way it decides to power-limit apps even when settings tell it not to. Looks like this current issue is just another area where Win11 is not that great.

**Bottom line:** *disable OSD if you have GPU Tweak III + GTX1660S:*
![изображение](https://github.com/Danily07/Translumo/assets/2913210/7d7dbd66-56b4-4c76-819a-51bfb66cda2d)

P.S. I went ahead and tested Translumo compatibility with MSI Afterburner/RivaTuner Server. Translumo works fine with those.
